client/Dockerfile.dev

```diff
@@ -0,0 +1,7 @@
+FROM node:14.11.0 AS client-development
+RUN mkdir /srv/client && chown node:node /srv/client
+WORKDIR /srv/client
+USER node
+RUN mkdir -p node_modules
+COPY --chown=node:node package.json package.json ./
+RUN npm install --silent
```

Comment:
please, add an empty line at the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline